lib/mpl_toolkits/tests/test_axes_grid.py

```diff
@@ -39,13 +39,9 @@ def test_imagegrid_cbar_mode_edge():
         ax3.imshow(np.abs(arr), cmap='jet')
         ax4.imshow(np.arctan2(arr.imag, arr.real), cmap='hsv')
 
-        # In each row/column, the "first" colorbars must be overwritten by the
-        # "second" ones.  To achieve this, clear out the axes first.
         for ax in grid:
             ax.cax.cla()
-            cb = ax.cax.colorbar(
-                ax.images[0],
-                ticks=mpl.ticker.MaxNLocator(5))  # old default locator.
+            cb = ax.cax.colorbar(ax.images[0])  # old default locator.
```

Comment:
Remove this comment, because you have removed the old locator
```suggestion
            cb = ax.cax.colorbar(ax.images[0])
```